pkg/gitops/profile.go

```diff
@@ -0,0 +1,138 @@
+package gitops
+
+import (
+	"context"
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
```

Comment:
Do you think it's the right kind of thing? If so, perhaps it would be good to open an issue and where we will enumerate which package have filesystem operation that need better testing and could benefit from something like aero?

Reply:
I learned from the other PR that this package helps us make the code testable by using an inMemory filesystem. Yes, I can definitely open a ticket about it.

Reply:
Done